token-upgrade/cli/src/main.rs

```diff
@@ -0,0 +1,831 @@
+                    .takes_value(true)
+                    .help("Specify the escrow account address to transfer from. [default: associated token account for the escrow authority]"),
+            )
+            .arg(
+                Arg::new("destination")
+                    .long("destination")
+                    .value_name("DESTINATION_ACCOUNT_ADDRESS")
+                    .validator(|s| is_valid_pubkey(s))
+                    .takes_value(true)
+                    .help("Specify the destination account to receive new tokens. [default: associated token account for owner on new mint]"),
+            )
+            .arg(
+                Arg::new("multisig_signer")
+                    .long("multisig-signer")
+                    .validator(|s| is_valid_signer(s))
+                    .value_name("MULTISIG_SIGNER")
+                    .takes_value(true)
+                    .multiple(true)
+                    .min_values(0)
+                    .max_values(spl_token_2022::instruction::MAX_SIGNERS)
+                    .help("Member signer of a multisig account")
+            )
+        )
+        .get_matches();
+
+    let (command, matches) = app_matches.subcommand().unwrap();
+    let mut wallet_manager: Option<Arc<RemoteWalletManager>> = None;
+
+    let config = {
+        let cli_config = if let Some(config_file) = matches.value_of("config_file") {
+            solana_cli_config::Config::load(config_file).unwrap_or_default()
+        } else {
+            solana_cli_config::Config::default()
+        };
+
+        let payer = DefaultSigner::new(
+            "payer",
+            matches
+                .value_of(&"payer")
+                .map(|s| s.to_string())
+                .unwrap_or_else(|| cli_config.keypair_path.clone()),
+        );
+
+        let json_rpc_url = normalize_to_url_if_moniker(
+            matches
+                .value_of("json_rpc_url")
+                .unwrap_or(&cli_config.json_rpc_url),
+        );
+
+        Config {
+            commitment_config: CommitmentConfig::confirmed(),
+            payer: Arc::from(
+                payer
+                    .signer_from_path(matches, &mut wallet_manager)
+                    .unwrap_or_else(|err| {
+                        eprintln!("error: {}", err);
+                        exit(1);
+                    }),
+            ),
+            json_rpc_url,
+            verbose: matches.is_present("verbose"),
+        }
+    };
+    solana_logger::setup_with_default("solana=info");
+
+    if config.verbose {
+        println!("JSON RPC URL: {}", config.json_rpc_url);
+    }
+    let rpc_client = Arc::new(RpcClient::new_with_commitment(
+        config.json_rpc_url.clone(),
+        config.commitment_config,
+    ));
+
+    match (command, matches) {
+        ("create-escrow", arg_matches) => {
+            let original_mint = pubkey_of(arg_matches, "original_mint").unwrap();
+            let new_mint = pubkey_of(arg_matches, "new_mint").unwrap();
```

Comment:
See above comments. These args ought to be required, if you're going to unwrap them.

Reply:
Yep, makes sense, thanks!